.github/workflows/gradle.yml

```diff
@@ -28,43 +27,31 @@ jobs:
          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
          sudo apt-get clean
          df -h
-      - uses: actions/checkout@v2
-      - uses: actions/cache@v2.1.4
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3.0.8
         with:
           path: ~/.gradle/caches
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
       - name: Set up JDK
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3.5.1
         with:
           distribution: 'adopt'
           java-version: ${{ matrix.java }}
       - name: Optional setup step
         run: |
           [ -f ./setup.sh ] && ./setup.sh || true
       - name: Build with Gradle
-        run: ./gradlew dependencyUpdates check --no-daemon --parallel --continue
+        run: ./gradlew build
         env:
-           TESTCONTAINERS_RYUK_DISABLED: true        
-      - name: Publish to Sonatype Snapshots
-        if: success() && github.event_name == 'push' && matrix.java == '8'
-        env:
-          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
-          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
-        run: ./gradlew publishToSonatype docs --no-daemon
-      - name: Determine docs target repository
-        uses: haya14busa/action-cond@v1
-        id: docs_target
+           TESTCONTAINERS_RYUK_DISABLED: true
+      - uses: ravsamhq/notify-slack-action@v2
+        if: always()
         with:
-          cond: ${{ github.repository == 'micronaut-projects/micronaut-core' }}
-          if_true: "micronaut-projects/micronaut-docs"
-          if_false: ${{ github.repository }}
-      - name: Publish to Github Pages
-        if: success() && github.event_name == 'push' && matrix.java == '8'
-        uses: micronaut-projects/github-pages-deploy-action@master
+          status: ${{ job.status }}
+          notification_title: "TEST - {workflow} has {status_message} - repo_url"
+          message_format: "TEST - {emoji} *{workflow}* {status_message} in <{repo_url}|{repo}@{branch}>"
+          notify_when: "success,failure,cancelled,warnings"
         env:
-          TARGET_REPOSITORY: ${{ steps.docs_target.outputs.value }}
-          GH_TOKEN: ${{ secrets.GH_TOKEN }}
-          BRANCH: gh-pages
-          FOLDER: build/docs
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
this should be `secrets.DA_SLACK_WEBHOOK_URL`

Reply:
That's done, thanks.